lib/account.js

```diff
@@ -13,6 +13,10 @@ function account (setupPromise, state, findAccountOptions) {
           return Promise.reject(errors.TOKEN_TYPE_INVALID)
         }
 
+        if (!tokenOptions.hasOwnProperty('timeout')) {
+          tokenOptions.timeout = 7200
+        }
```

Comment:
I’m sorry I should have thought about this earlier, but can we add another timestamp `expiresAt` instead of adding a `timeout` property? It will be simpler to lookup later as we can directly look for that property without any calculation. Or do you see any downside with that approach?